libs/sandboxes-service-api-client/tests/Sandboxes/SandboxesApiClientTest.php

```diff
@@ -47,7 +54,10 @@ public function testGetSandbox(): void
         );
         $result = $client->getSandbox('sandbox-id');
 
-        self::assertEquals($responseBody, $result);
+        // phpcs:ignore Generic.Files.LineLength
+        unset($responseBody['projectId'], $responseBody['tokenId'], $responseBody['componentId'], $responseBody['configurationVersion']);
+
+        self::assertEquals($responseBody, $result->toArray());
```

Comment:
Nez porovnavat pole ,kde musis neco unsetovat = vedet, jak uvnitr funguje `Sandbox::toArray()`, coz neni predmetem tohohle testu, nebylo by lepsi porovnat

```suggestion
        self::assertEquals(Sandbox::fromArray($responseBody), $result);
```

Reply:
To by platilo samozrejme i pro ostatni testy

Reply:
To je sakra pravda. 🙂 Upraveno.